vanilla-framework/vanilla-framework
# Add a setting for navigation background

## Done

Add a setting for navigation background color

## QA

- Pull code
- Run `./run serve --watch`
- Open and check nothing broken on:
http://0.0.0.0:8101/vanilla-framework/examples/patterns/navigation/default/
http://0.0.0.0:8101/vanilla-framework/examples/patterns/navigation/dark/
http://0.0.0.0:8101/vanilla-framework/examples/patterns/navigation/light/

## Details

Fixes #1354
